xml/article_kernel_tasks.xml

```diff
@@ -0,0 +1,227 @@
+    waiting for an external signal but it is still able to interact with the
+    userspace via signals.</para>
+   <para>The task state is visible from the userspace.</para>
+  </listitem>
+ </varlistentry>
+ 
+ <varlistentry>
+  <term>
+   <parameter>TASK_UNINTERUPTIBLE</parameter>
+  </term>
+  <listitem>
+   <para>
+    Task is blocked while waiting for an event or condition, as for example
+    <parameter>mutex</parameter>. Any userspace interaction is deferred during
+    this time. Tasks blocked in this state, contribute to the load average. If
+    this tasks is blocked for too long, it can get reported by the hung task
+    detector.
+    </para>
+   <para>The task state is visible from the userspace.</para>
+  </listitem>
+ </varlistentry>
+ 
+ <varlistentry>
+  <term>
+   <parameter>TASK_KILLABLE</parameter>
+  </term>
+  <listitem>
+   <para>
+    Task is uninterruptible put to sleep which allows to wake up on a fatal
+    signal, e.g. <parameter>SIGKILL</parameter>. The task contributes to the
+    load average and is not considered for hung task detectors.
+   </para>
+   <para>
+    <parameter>TASK_KILLABLE</parameter> is used when waiting for an
+    <parameter>I/O</parameter> operation on NFS mount, but it allows to kill
+    the waiting application in case the operation takes too long.
+   </para>
+   <para>The task state is not visible from the userspace.</para>
+  </listitem>
+ </varlistentry>
+ 
+ <varlistentry>
+ <term>
+  <parameter>TASK_IDLE</parameter>
+ </term>
+ <listitem>
+  <para>
+   Task is uninterruptible put to sleep until a condition evaluates to
+   <parameter>TRUE</parameter>. This task does not contribute to the load
+   average and is not reported by the hung task detector.
+  </para>
+  <para>
+   <parameter>TASK_IDLE</parameter> is used for tasks that are expected to
+   be blocked for a long time, as for example with
+   <parameter>schedule_timeout_idle</parameter>.
+  </para>
+  <para>The task state is not visible from the userspace.</para>
+ </listitem>
+ </varlistentry>
+ 
+ <varlistentry>
+  <term>
+   <parameter>TASK_STOPPED</parameter>
+  </term>
+  <listitem>
+   <para>
+    Task has received a <parameter>SIGSTOP</parameter> from userspace and is
+    blocked until it gets <parameter>SIGCONT</parameter>.
+   </para>
+   <para>The task state is visible from the userspace.</para>
+  </listitem>
+ </varlistentry>
+ 
+ <varlistentry>
+  <term>
+   <parameter>TASK_TRACED</parameter>
+  </term>
+  <listitem>
+   <para>
+    Task is blocked during <parameter>ptrace</parameter>. For more information
+    on <parameter>ptrace</parameter>, see the man pages with
+    <command>man ptrace(2)</command>.
+   </para>
+   <para>The task state is visible from the userspace.</para>
+  </listitem>
+ </varlistentry>
+ 
+ <varlistentry>
+  <term>
+   <parameter>TASK_PARKED</parameter>
+  </term>
+  <listitem>
+   <para>
+    A kernel thread is parked.
```

Comment:
Could it found a nice parking slot? :wink: Ok, seriously, I still don't know what that means. The parameter has "TASK" in its name, but you mention thread. I'm a bit confused.